patches/net/minecraft/client/renderer/blockentity/SkullBlockRenderer.java.patch

```diff
@@ -19,7 +19,7 @@
 +        SkullBlock.Type type = ((AbstractSkullBlock) blockEntity.getBlockState().getBlock()).getType();
 +        if (type == SkullBlock.Types.DRAGON) {
 +            net.minecraft.core.BlockPos pos = blockEntity.getBlockPos();
-+            return new net.minecraft.world.phys.AABB(pos.getX() - .5, pos.getY(), pos.getZ() - .5, pos.getX() + 1.5, pos.getY() + 1.5, pos.getZ() + 1.5);
++            return new net.minecraft.world.phys.AABB(pos.getX() - .625, pos.getY() - .3125, pos.getZ() - .625, pos.getX() + 1.62, pos.getY() + 1.0, pos.getZ() + 1.625);
```

Comment:
Why is xMax `1.62` and zMax `1.625`? If we are worried about potentially getting the numbers wrong I wouldn't mind rounding to full block coordinates just in case, BTW.